ios/bottom-tabs/RNSBottomTabsHostComponentView.mm

```diff
@@ -266,6 +266,26 @@ - (void)updateProps:(const facebook::react::Props::Shared &)props
       }
   }
 
+  if (newComponentProps.tabBarControllerMode != oldComponentProps.tabBarControllerMode) {
+#if RNS_IPHONE_OS_VERSION_AVAILABLE(18_0)
+    if (@available(iOS 18.0, *)) {
+      _tabBarControllerMode = rnscreens::conversion::UITabBarControllerModeFromRNSBottomTabsTabBarControllerMode(
+          newComponentProps.tabBarControllerMode);
+      _controller.mode = _tabBarControllerMode;
+      if (_tabBarControllerMode == UITabBarControllerModeTabSidebar) {
+        // Also show the sidebar on iPad if the device is in landscape mode (default behavior)
+        UIDeviceOrientation current = [[UIDevice currentDevice] orientation];
+        if (UIDeviceOrientationIsLandscape(current) ) {
+          _controller.sidebar.hidden = false;
+        }
```

Comment:
If this is UIKit default, why do we enforce it manually? 

I also don't like the code using device orientation here, since it does not have to match the effective interface orientation of the application. This is not a blocker, just noting to take a second look after we land this PR.

Reply:
I see in Apple docs that this should indeed be the [default]:(https://developer.apple.com/documentation/uikit/elevating-your-ipad-app-with-a-tab-bar-and-sidebar?language=objc#Integrate-the-tab-bar-and-sidebar) 
<img width="1885" height="172" alt="image" src="https://github.com/user-attachments/assets/21cee2a4-1157-4a0b-9219-1b187aab0b57" />

however, do we have an idea why isn't it applied out-of-the-box & we need to do it manually?

Reply:
Testing the runtime again & it seems to indeed work w/o this code you added here (I could swear that first time I tested it, it didn't work)

https://github.com/user-attachments/assets/373f0423-ebd6-4001-a554-fd32c2945ad3

Why is this code needed then?